pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -651,6 +654,7 @@ public RebalanceResult rebalance(
     rebalanceConfig.setPreChecks(preChecks);
     rebalanceConfig.setReassignInstances(reassignInstances);
     rebalanceConfig.setIncludeConsuming(includeConsuming);
+    rebalanceConfig.setMinimizeDataMovement(minimizeDataMovement);
```

Comment:
Can the conversion of this String to `Boolean` be done here itself? i.e. the parsing for TRUE/FALSE and DEFAULT